### PR TITLE
Add log-level option to cli

### PR DESCRIPTION
**Pull request by 6543**

### CODE REVIEW

File: cli/build/build_approve.go [hunk 1]
```diff
@@ -19,6 +19,7 @@ var buildApproveCmd = &cli.Command{
 }
 
 func buildApprove(c *cli.Context) (err error) {
+	common.SetupConsoleLogger(c)
```
Comment (anbraten) [inline cli/build/build_approve.go:22]: Isn't there something like a global hook / init fnc for `github.com/urfave/cli/v2` to call this logger setup?

**Review by anbraten — APPROVED** [review]

File: cli/build/build_approve.go (hunk 1 above)
Comment (anbraten) [reply cli/build/build_approve.go:22]: from checking the docs there only seems to be an option for commands called `Before`:

```go
       Before: func(c *cli.Context) error {
          fmt.Fprintf(c.App.Writer, "brace for impact\n")
          return nil
        },
```

But I doesn't look like we could save code that way.

Comment (anbraten) [reply cli/build/build_approve.go:22]: Ohh, from the api it seems as we could add it to the app as global before hook as well.

Comment (6543) [reply cli/build/build_approve.go:22]: I'll have a look - that's why it's not merged jet :D

Comment (anbraten) [reply cli/build/build_approve.go:22]: Yeah no pressure. Just wanted to inform you about my investigations on it. 😉

Comment (6543) [reply cli/build/build_approve.go]: Before did the job ...

**Review by anbraten — APPROVED** [review]
nice. the new change looks very clean and DRY now

**Pull request merged**